src/picknik_ur_site_config/objectives/take_snapshot.xml

```diff
@@ -5,7 +5,11 @@
         <Control ID="Sequence">
             <Action ID="GetPointCloud" topic_name="/wrist_mounted_camera/depth/color/points" message_out="{point_cloud}" />
             <Action ID="UpdatePlanningSceneService" point_cloud="{point_cloud}" point_cloud_service="/point_cloud_service"/>
-            <Action ID="GetStringFromUser" parameter_name="take_snapshot.uuid" parameter_value="{uuid}" />  
+            <Control ID="Fallback" name="TryGetStringFromUser">
+                <Action ID="GetStringFromUser" parameter_name="take_snapshot.uuid" parameter_value="{uuid}" />
+                <!-- if we fail to get the string from the user fallback to an empty value for the uuid -->
+                <Action ID="Script" code="uuid = ''"/>
```

Comment:
Should it be

```suggestion
                <Action ID="Script" code="uuid := ''"/>
```

because this may create a new blackboard entry if it doesn't exist, per [the BT.CPP docs](https://www.behaviortree.dev/docs/guides/scripting).

Reply:
Good catch but it also works without the `:` for some reason.